docs/docs/guides/labs/components/creating-new-component-types/creating-dg-plugin.md

```diff
@@ -0,0 +1,91 @@
+---
+title: 'Creating a `dg` plugin'
+sidebar_position: 200
+---
+
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+A `dg` plugin is a Python package that defines and exposes `dg`-legible
```

Comment:
A `dg` plugin is a Python package that defines and exposes `dg`-legible classes and functions (which we will refer to as "objects")